nautobot_ssot/jobs/base.py

```diff
@@ -47,17 +48,56 @@ class DataSyncBaseJob(BaseJob):
 
     dry_run = BooleanVar()
 
+    def load_source_adapter(self):
+        """TODO:."""
+        pass
+
+    def load_target_adapter(self):
+        """TODO:."""
+        pass
+
+    def calculate_diff(self):
+        """TODO:."""
+        pass
+
+    def execute_sync(self):
+        """TODO:."""
+        pass
```

Comment:
I think we should be able to provide default implementations of at least `calculate_diff` and `execute_sync`, as in many cases concrete subclasses shouldn't need to override these.

Reply:
Could we provide some sane default for all of them ?

Reply:
I think so, the one from the example for instance